src/components/Articles/CompanyProvisions.vue

```diff
@@ -0,0 +1,227 @@
+                </v-list-item-subtitle>
+              </v-list-item>
+            </v-list>
+          </v-menu>
+        </span>
+      </v-flex>
+    </v-layout>
+    <v-layout v-else>
+      <v-flex xs3>
+        <label><strong>Pre-existing Company Provisions</strong></label>
+      </v-flex>
+      <v-flex xs9>
+        <v-layout>
+          <v-flex>
+            <p class="instructions-paragraph">
+              Complete this item only if the company has resolved that none of the Pre-existing Company Provisions
+              are to apply to this company (refer to Part 17 and Table 3 of the Regulation under the Business
+              Corporations Act).
+            </p>
+            <v-layout>
+                <v-checkbox
+                  id="cp-checkbox"
+                  class="mt-1"
+                  :color="isInvalid ? checkBoxColor : ''"
+                  v-model="draftProvisionsRemoved"
+                />
+                <span class="checkbox-label" :class="{ 'invalid': isInvalid }">
+                  The company has resolved that none of the Pre-existing Company Provisions are to apply to this company
+                </span>
+            </v-layout>
+          </v-flex>
+        </v-layout>
+        <v-layout pt-10>
+          <v-flex xs12>
+            <div class="action-btns">
+              <v-btn
+                large
+                color="primary"
+                id="company-provisions-done"
+                @click="setCompanyProvisionsDone()"
+              >
+                <span>Done</span>
+              </v-btn>
+              <v-btn
+                large
+                outlined
+                color="primary"
+                id="company-provisions-cancel"
+                @click="cancelCompanyProvisionChange()"
+              >
+                <span>Cancel</span>
+              </v-btn>
+            </div>
+          </v-flex>
+        </v-layout>
+      </v-flex>
+    </v-layout>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Mixins, Watch } from 'vue-property-decorator'
+import { CommonMixin } from '@/mixins'
+import { ConfirmDialog } from '@/components/dialogs'
+
+@Component({
+  components: {
+    ConfirmDialog
+  }
+})
+export default class CompanyProvisions extends Mixins(CommonMixin) {
+  private draftProvisionsRemoved: boolean = false
+  private isEditing: boolean = false
+  private haveChanges: boolean = false
+  private originalProvisionsRemovedValue: boolean = false
+  private isInvalid = false
+  private checkBoxColor = '#d3272c' // $app-red
+
+  // Props
+  @Prop({ default: () => { return false } })
+  private provisionsRemoved!: boolean
+
+  // Emitters
+  @Emit('companyProvisionsChanged')
+  private emitCompanyProvisionsChanged (provisionsremoved: boolean): void {}
+
+  @Emit('haveChanges')
+  private emitHaveChanges (haveChanges: boolean): void {}
+
+  // Watchers
+  @Watch('provisionsRemoved', { deep: true, immediate: true })
```

Comment:
You don't need "deep" as this is not an object (ie, don't need to watch object props).

Reply:
Ditto on line 147.

Reply:
yes, you're right. Thanks!

Reply:
Done.